docs/configuration_syntax.md

```diff
@@ -128,6 +128,18 @@ pull:
     # discovered project refs (optional, default: 30)
     interval_seconds: 30
 
+  metrics_with_traces:
+    # Interval in seconds to pull trace match metrics from
+    # discovered project refs (optional, default: 1800)
+    interval_seconds: 1800
+
+  trace_rules:
```

Comment:
I would omit this parameter in here and use the structure directly within the project settings to avoid confusion